modules/mono/editor/GodotTools/GodotTools/Utils/OS.cs

```diff
@@ -70,7 +70,7 @@ public static class DotNetOS
         {
             ["Windows"] = Platforms.Windows,
             ["macOS"] = Platforms.MacOS,
-            ["LinuxBSD"] = Platforms.LinuxBSD,
+            ["Linux"] = Platforms.LinuxBSD,
```

Comment:
So I assume this was throwing `Target platform not supported`, right?

Reply:
Yeah